### DISCUSSION TITLE
solve independent systems of equations in parallel

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [libMesh/libmesh#3753] [Q&A]
How to achieve parallel (simultaneous) solving of a series of au ''=f? (Each equation only has different input parameters ‘a’ and ‘f’. If possible, can you provide a simple reference example?)? thank you very much.

**Comment by lindsayad** [comment]
This seems like a duplicate of https://github.com/libMesh/libmesh/discussions/3735
